walkthroughs/tls-with-acm/infrastructure/ecs-service.yaml

```diff
@@ -275,8 +237,9 @@ Resources:
             - 'Fn::ImportValue': !Sub "${EnvironmentName}:PrivateSubnet1"
             - 'Fn::ImportValue': !Sub "${EnvironmentName}:PrivateSubnet2"
       TaskDefinition: { Ref: ColorGatewayTaskDefinition }
+      HealthCheckGracePeriodSeconds: 3000
```

Comment:
50 minutes of grace period? It should stabilize pretty quickly.

Reply:
Yeah, this is a typo, 300 secs should be ok here.